pkg/cli/validate.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/abcxyz/jvs/client-lib/go/client"
+	"github.com/spf13/cobra"
+)
+
+var (
+	flagToken           string
+	flagAllowBreakglass bool
```

Comment:
I wonder if we need this to be a flag. We should always allow breakglass but print out a warning if it's a breakglass token.

We can check if it's a breakglass token with https://github.com/abcxyz/jvs/blob/main/apis/v0/breakglass.go#L64

Alternatively, we could use such a flag to control whether to return an error but we should still parse the breakglass token and output its content.

Reply:
I think I would expect `jvsctl validate` to print out all information about the token, including whether it's breakglass. It should exit 1 if validation fails and exit 0 if validation succeeds. But either way, I'd expect to see `breakglass: true|false` in the output.